ports/mimxrt/modmachine.h

```diff
@@ -35,9 +35,15 @@ extern const mp_obj_type_t machine_rtc_type;
 extern const mp_obj_type_t machine_i2c_type;
 extern const mp_obj_type_t machine_spi_type;
 extern const mp_obj_type_t machine_uart_type;
+#if USDHC_AVAIL
```

Comment:
don't really need this `#if`; the type just won't exist (at the linker stage) if it's not enabled

Reply:
fixed.